### PR TITLE
DSD-1152: Close Chakra's Tooltip

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-1152](https://jira.nypl.org/browse/DSD-1152)

## This PR does the following:

- Adds more props to Chakra's `Tooltip` component so that it closes more forcefully and does not stay open, especially on DS `Image`s.

## How has this been tested?

Locally and in Turbine. After speaking with Deepika on this, the `Tooltip` does _eventually_ close if it's hovered on and off again. So it doesn't stay on indefinitely. The recommendation would be to not use the `shouldWrapChildren` prop for DS `Image`s so they render correctly.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -58,12 +58,16 @@ export const Tooltip = chakra(
 
     return (
       <ChakraTooltip
-        hasArrow
         aria-label={typeof content !== "string" ? "Tooltip" : undefined}
-        label={content}
+        closeDelay={750}
```
Comment (bigfishdesign13) [inline src/components/Tooltip/Tooltip.tsx:62]: Does the addition of these new props act as a backup to ensure that the tooltip does close eventually?  Meaning, does the bug still exist and we are just adding something to gracefully get around the bug?  Or does this resolve the bug altogether?

**Review by bigfishdesign13 — COMMENTED** [review]
Please see my question.  I am just curious before I approve.

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/Tooltip/Tooltip.tsx:62]: Yep, pretty much works as a backup to ensure it closes. That being said, it does resolve the bug but in a roundabout way.

**Review by bigfishdesign13 — APPROVED** [review]
Release the hounds!

**Pull request merged**